src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-export/rgw-multisite-export.component.html

```diff
@@ -0,0 +1,60 @@
+<cd-modal [modalRef]="activeModal">
+  <ng-container i18n="form title"
+                class="modal-title">Export Multi-site Realm Token</ng-container>
+
+  <ng-container class="modal-content">
+    <form name="exportTokenForm"
+          #frm="ngForm"
+          [formGroup]="exportTokenForm">
+    <span *ngIf="loading"
+          class="d-flex justify-content-center">
+    <i [ngClass]="[icons.large3x, icons.spinner, icons.spin]"></i></span>
+    <div class="modal-body"
+         *ngIf="!loading">
+      <cd-alert-panel *ngIf="!tokenValid"
+                      type="warning"
+                      class="mx-3"
+                      i18n>
+      <div *ngFor="let realminfo of realms">
+        <b>{{realminfo.realm}}</b> -
+        {{realminfo.token}}
+      </div>
+      </cd-alert-panel>
+      <div *ngFor="let realminfo of realms; let i = index">
+        <div class="form-group row">
+          <label class="cd-col-form-label"
+                 for="realmName"
+                 i18n>{{ i+1 }}. Realm Name:
```

Comment:
i am not fond of how this is shown. Let's revert it back to the earlier design for now. We could think about a better UX after this PR.

Reply:
Reverted to the previous one